### ISSUE TITLE
[resolver-webpack] Compat with multiple configs

### ISSUE THREAD

**Issue by pitaj** [benmosher/eslint-plugin-import#181]
`webpack.config.js` can be an array of webpack config objects like so:

```js
module.exports = [
  {
    name: 'configurator',
  },
  {
    name: 'scene',
  }
];
```

Currently, I have to append this to my config file:

```js
module.exports.resolve = {
  extensions: [
    '',
    '.jsx',
    '.js',
  ],
};
```

... which shouldn't be necessary. Please add support for multiple webpack configs.

**Comment by benmosher** [comment]
Makes sense. Will look into it. 

FWIW, if the groups of files covered by each config are fairly disjoint--though I suspect they aren't--you could use nested `.eslintrc`s and set one config for one set and another for another.

**Comment by benmosher** [comment]
Scratch that suggestion. Just realized you meant multiple configs in a single file.

**Comment by mk0x9** [comment]
Specify list of extensions in `.eslintrc`?

**Comment by benmosher** [comment]
Actually, might be simpler if you create a `webpack.base.js` with `extensions`, `resolve`, etc. and reference it from the multi-config.

You may then specify that base config as the webpack config for linting.

Otherwise, it may be difficult to determine which of the multiple configs is applicable to the file being linted.

Happy to take a PR if someone has a thought on how to do this without making potentially faulty assumptions.

**Comment by GreenGremlin** [comment]
Here's a PR: https://github.com/benmosher/eslint-plugin-import/pull/335

**Comment by TheSeally** [comment]
@benmosher but if I have aliases in both configs what I should do? Is it possible to merge resolve sections from all configs?
I can try to make PR with config aliases merge if there are no problem conceptually
